test/fail_compilation/checkimports3.d

```diff
@@ -0,0 +1,15 @@
+/*
+REQUIRED_ARGS: -transition=checkimports -de
+TEST_OUTPUT:
+---
+fail_compilation/checkimports3.d(14): Deprecation: local import search method found overloadset checkimports3.foo (3 overloads) instead of overloadset checkimports3.foo (2 overloads)
```

Comment:
You can't really fix the deprecations, as long as you pass -transition=checkimports the compiler lists differences between the old and new lookup/import rules that must be inspected manually.
Yes `local import search method` is a weird name, that's the new lookup rule, though the error message works vise versa. Still fixing the message isn't part of this PR.